tests/manage/pv_services/test_raw_block_pv.py

```diff
@@ -0,0 +1,165 @@
+            )
+
+        pvs = [pvc.backed_pv_obj for pvc in pvcs]
+
+        pod_dict = constants.CSI_RBD_RAW_BLOCK_POD_YAML
+        pvc_mb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_mb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes)
+        )
+        ) for _ in range(3)
+        ]
+
+        pvc_gb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_gb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes)
+        )
+        ) for _ in range(3)
+        ]
+
+        pvc_tb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_tb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes
+            )
+        )
+        ) for _ in range(3)
+        ]
+
+        def flatten(l):
+            return [item for sublist in l for item in sublist]
+        pods = [pvc_mb_pods, pvc_gb_pods, pvc_tb_pods]
+        pods = flatten(pods)
+        for pod in pods:
+            helpers.wait_for_resource_state(
+                resource=pod, state=constants.STATUS_RUNNING, timeout=120)
+        storage_type = 'block'
+
+        with ThreadPoolExecutor() as p:
+            for pod in pvc_mb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(10,200)}M',
+                )
+            for pod in pvc_gb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(1,5)}G',
+                )
+            for pod in pvc_tb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(10,15)}G',
+                )
+
+        for pod in pods:
+            get_fio_rw_iops(pod)
+        return pods, pvcs, pvs
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-750")
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+)
+class TestRawBlockPVRetain(BaseRawBlockPV):
+    reclaim_policy = constants.RECLAIM_POLICY_RETAIN
+
+    def test_raw_block_pv(self, teardown_factory):
+        pods, pvcs, pvs = self.raw_block_pv()
+        teardown_factory(pvs)
+        teardown_factory(pvcs)
+        teardown_factory(pods)
+
+
+@bugzilla('1726266')
```

Comment:
not sure if we should block this test from running due to this bug.
Now, that we are validation PV deletion in pvc_factory, is the PV still exists as a leftover?

Reply:
It takes more time but gets deleted. Will remove bugzilla mark